cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/ListsTest.scala

```diff
@@ -27,149 +27,145 @@ class ListsTest extends DocumentingTest {
   override def doc = new DocBuilder {
     doc("Lists", "cypher-lists")
     initQueries(
-      """CREATE (keanu:Person {name: 'Keanu Reeves'}),
-                |(johnnymnemonic:Movie {title: 'Johnny Mnemonic', released: 1995}),
-                |(somethingsgottagive:Movie {title: 'Somethings Gotta Give', released: 2003}),
-                |(thematrixrevolutions:Movie {title: 'The Matrix Revolutions', released: 2003}),
-                |(thematrixreloaded:Movie {title: 'The Matrix Reloaded', released: 2003}),
-                |(thereplacements:Movie {title: 'The Replacements', released: 2000}),
-                |(thematrix:Movie {title: 'The Matrix', released: 1999}),
-                |(thedevilsadvocate:Movie {title: 'The Devils Advocate', released: 1997}),
-                |
-                |(keanu)-[:ACTED_IN]->(johnnymnemonic),
-                |(keanu)-[:ACTED_IN]->(somethingsgottagive),
-                |(keanu)-[:ACTED_IN]->(thematrixrevolutions),
-                |(keanu)-[:ACTED_IN]->(thematrixreloaded),
-                |(keanu)-[:ACTED_IN]->(thereplacements),
-                |(keanu)-[:ACTED_IN]->(thematrix),
-                |(keanu)-[:ACTED_IN]->(thedevilsadvocate)
-
-      """.stripMargin)
+      """CREATE
+        #  (keanu:Person {name: 'Keanu Reeves'}),
+        #  (johnnymnemonic:Movie {title: 'Johnny Mnemonic', released: 1995}),
+        #  (somethingsgottagive:Movie {title: 'Somethings Gotta Give', released: 2003}),
```

Comment:
```suggestion
        #  (somethingsgottagive:Movie {title: 'Something's Gotta Give', released: 2003}),
```
If the apostrophe doesn't mean trouble, it is the correct title of the movie so please keep.